binding/lua/multiverso.lua

```diff
@@ -113,27 +113,27 @@ function mv.MatrixTableHandler:get(row_ids)
     if row_ids == nil then
         cdata = ffi.new("float[?]", self._size)
         libmv.MV_GetMatrixTableAll(self._handler[0], cdata, self._size)
-        data = util.cdata2array(cdata, tonumber(self._size))
+        data = util.cdata2tensor(cdata, tonumber(self._size))
         return torch.reshape(data, tonumber(self._num_row), tonumber(self._num_col))
     else
-        crow_ids = util.array2cdata(row_ids, #row_ids, "int[?]")
+        crow_ids = util.tensor2cdata(row_ids, 'int')
         crow_ids_n = ffi.new("int", #row_ids)
         cdata = ffi.new("float*[?]", #row_ids * self._num_col)
         lib.MV_GetMatrixTableByRows(self._handler[0], crow_ids, crow_ids_n, self._num_col, cdata)
-        data = util.cdata2array(cdata, tonumber(self._size))
+        data = util.cdata2tensor(cdata, tonumber(self._size))
```

Comment:
Yep, you are right. Sorry for the silly error.

I suddenly realize that we don't have test for the partial get. I will come back with more robust unittest.